### PR TITLE
[SKULL-16] Main page & Rules

### PR DESCRIPTION
**Pull request by siwoo-jung**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,21 +1,31 @@
 import React from "react";
+import { Routes, Route } from "react-router-dom";
```
Comment (SkellyBG) [inline src/App.tsx:2]: remember to `yarn add react-router-dom` and commit the change to package.json and yarn.lock!

File: src/components/MainPage.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <div className="logo-container">
+        <img src="src/images/bee.png"></img>
+        <img src="src/images/hive_logo.png"></img>
+        <img src="src/images/logo2.png"></img>
+      </div>
+      <div className="btn-container">
+        <button className="btn" id="btn-play">
+          <Link to="/play" id="btn-play-link">
+            PLAY
+          </Link>
```
Comment (SkellyBG) [inline src/components/MainPage.tsx]: `<Link>` component should wrap outside the button, not the text

File: src/App.tsx [hunk 3]
```diff
@@ -1,21 +1,31 @@
...
+    //     justifyContent: "center",
+    //     alignItems: "center",
+    //   }}
+    // >
+    //   <GridBoard />
+    // </div>
+    <>
+      <Routes>
+        <Route path="/" element={<MainPage />} />
+        <Route path="/rules" element={<Rules />} />
```
Comment (SkellyBG) [inline src/App.tsx]: the Route for `/play` is missing - for now, we can put the commented out code in a separate component and link the route to that

File: src/App.tsx [hunk 4]
```diff
@@ -1,21 +1,31 @@
...
+    //     height: "100vh",
+    //     display: "flex",
+    //     flexDirection: "column",
+    //     justifyContent: "center",
+    //     alignItems: "center",
+    //   }}
+    // >
+    //   <GridBoard />
+    // </div>
+    <>
```
Comment (SkellyBG) [inline src/App.tsx]: don't need a fragment here

File: src/App.tsx (hunk 1 above)
Comment (siwoo-jung) [reply src/App.tsx:2]: Did `yarn add react-router-do` and did another PR

File: src/components/MainPage.tsx (hunk 2 above)
Comment (siwoo-jung) [reply src/components/MainPage.tsx]: Fixed

File: src/App.tsx (hunk 4 above)
Comment (siwoo-jung) [reply src/App.tsx]: Fixed

File: package-lock.json
Comment (SkellyBG) [inline package-lock.json:1]: we're using yarn - there shouldn't be a package-lock.json file

Comment (SkellyBG) [reply package-lock.json:1]: to use yarn, do `npm install -g yarn`

**Comment by SkellyBG** [comment]
merging so changes can be made upstream

**Pull request merged**